<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningDialog</class>
 <widget class="QWidget" name="WarningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>991</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;border:none;border-top-left-radius: 8px;border-top-right-radius:8px}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>164</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_Label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>800</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备已急停，请将急停复位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gameTips_Label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>600</width>
      <height>158</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_Label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px 8px 0px 0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>17</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/release/source/image/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
